Source/Fuse.Common/Profiling.uno

```diff
@@ -334,21 +245,20 @@ namespace Fuse
 			Write(id);
 		}
 
+		static string[] _cachedStrings = new string[0xFF];
+
 		private static byte CacheString(string str)
 		{
-			var isCached = _stringCache.IsStringCached(str);
-			var id = _stringCache.CacheString(str);
-
-			if (isCached)
-				return id;
-
-			debug_log("Caching string to remote: " + str);
-
-			Write(Command.CacheString);
-			Write(id);
-			Write(str);
+			var hash = (byte)(str.GetHashCode() & 0xFF);
+			if (_cachedStrings[hash] != str)
+			{
+				Write(Command.CacheString);
+				Write(hash);
+				Write(str);
+				_cachedStrings[hash] = str;
```

Comment:
This will fail in the case where `str.GetHashCode() & 0xFF == 0xFF`, since _cachedStrings is 255 elements big, as opposed to 256.